src/Grok.Net.Tests/UnitTests.cs

```diff
@@ -81,7 +81,21 @@ public void EmailPatternTest()
             Assert.Equal("Free as in Free Beer", grokResult[1].Value);
             Assert.Equal("email", grokResult[0].Key);
             Assert.Equal("comment", grokResult[1].Key);
-
         }            
+
+        [Fact]
+        public void IPv4PatternTest()
+        {
+            Grok act = new Grok("%{IPV4:IP}:%{GREEDYDATA:comment}");
+            string logs = @"172.26.34.32:Free as in Free Beer
+                10.0.12.17:In for the win";
+            
+            GrokResult grokResult = act.Parse(logs);
+           
+            Assert.Equal("172.26.34.32", grokResult[0].Value);
```

Comment:
Could you please declare string constants for better reuse? Like

```csharp
// Arrange
const string ip = "172.26.34.32";
const string comment = "Free as in Free Beer"; 
string logs = $"{ip}:{comment}";

// Act
...
// Assert
Assert.Equal(ip, grokResult[0].Value);
Assert.Equal(comment, grokResult[1].Value);
```

Reply:
Sure thing.

Reply:
Done.

Reply:
it is not done. where is const?

Reply:
Changes made as requested.